ui-tests/src/test/resources/features/customizations/connectors/apicurio.feature

```diff
@@ -91,7 +90,13 @@ Feature: Customization - API Connector - ApicurIO GUI
     And navigate to the "Connections" page
     And click on the "Create Connection" link
     And select "Kie Server API" connection type
-    Then check that connection authentication type has 1 options and contains text "No Security"
+#    Then check that connection authentication type has 1 options and contains text "No Security"
```

Comment:
Could these commented out steps be removed?

Reply:
ok.